app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,6 @@
+class DashboardController < ApplicationController
+  def index
+    @job     = Job.level_of_interest
```

Comment:
i suggest naming this `@jobs_by_level_of_interest` to be more explicit and the below `@companies_by_level_of_interest`